doc/howto/dqlite_troubleshoot.md

```diff
@@ -0,0 +1,85 @@
+# How to troubleshoot (some) dqlite errors
+
+dqlite is the distributed database that LXD uses to store information
+that must be synchronized across a cluster. This page describes how
+to respond if LXD fails to start or crashes because of a dqlite-related error.
+
+## Recognizing dqlite-related errors
+
+Look for error messages that mention keywords like `dqlite`, `raft`, or
+`segment`, especially if LXD is failing to start up.
```

Comment:
```suggestion
If LXD fails to start up or crashes, you should suspect a Dqlite-related error if the error message mentions keywords like `dqlite`, `raft`, or `segment`.
```

Reply:
Moving the symptoms (failing to start or crashing) into this section from the intro.